impeller/entity/contents/solid_color_contents.cc

```diff
@@ -44,6 +44,8 @@ std::optional<Rect> SolidColorContents::GetCoverage(
 bool SolidColorContents::Render(const ContentContext& renderer,
                                 const Entity& entity,
                                 RenderPass& pass) const {
+  auto capture = entity.GetCapture().CreateChild("SolidColorContents");
```

Comment:
Can we IFDEF these out?

Reply:
There's theoretically no need. All this stuff is inlined and should flatten out to zero instructions when the capture is build-time disabled. (I haven't rigorously proven this, mind you)

Reply:
Well, I'll see it in the profile if its there  😈  - but yeah, no need to worry about it until its a problem !

Reply:
I'm gonna Volkswagen the heck out of our benchmarks, but in a benign way.

Reply:
Just add some speedup loops